lang/java/reef-runtime-hdinsight/src/main/java/org/apache/reef/runtime/hdinsight/client/yarnrest/LocalResource.java

```diff
@@ -47,8 +47,6 @@
   private long size;
   private long timestamp;
 
-  private String url;
```

Comment:
Was this url added by you? If not, why are we modifying the hdinsight runtime?

Reply:
Oh, this is undoing some changes I made last week in a mad rush to get this off the ground and get the team developing driver code.